docs/Users_Guide/release-notes.rst

```diff
@@ -76,6 +76,350 @@ METplus Version 6.0.0 Beta 1 Release Notes (2023-09-15)
+
+Starting in v6.0.0, the deprecated environment variables are no longer set and
+an error message will be displayed for each deprecated variable that was found::
+
+    ERROR: Deprecated environment variables found in MET config file: /path/to/GridStatConfig_trey
+    ERROR: Deprecated environment variable ${MODEL} found
+    ERROR: Deprecated environment variable ${OBTYPE} found
+    ERROR: Deprecated environment variable ${REGRID_TO_GRID} found
+
+If these errors occur,
+the use case will not run until the METplus configuration file has been updated.
+
+How to upgrade
+^^^^^^^^^^^^^^
+
+Removing **<WRAPPER_NAME>_CONFIG_FILE**, e.g. :term:`GRID_STAT_CONFIG_FILE`,
+from the METplus config file will prevent the errors and
+allow the use case to run.
+However, this alone may result in changes to the output because the settings
+in the user-defined wrapped MET config file may no longer be set.
+
+**It is important to carefully review the settings and set the appropriate
+METplus configuration variables to preserve the original configuration!**
+
+Compare the user-defined wrapped MET config file (:term:`GRID_STAT_CONFIG_FILE`)
+with the default config file that is found in the MET installation location,
+e.g. share/met/GridStatConfig_default.
+The paths to the files to compare are listed in the *ERROR* log that follows
+the *ERROR* logs that list the deprecated environment values that were found.
+The errors also note the METplus config variable that will be removed,
+e.g. :term:`GRID_STAT_CONFIG_FILE`.
+
+::
+
+    ERROR: Deprecated environment variables found in MET config file: /path/to/GridStatConfig_trey
+    ERROR: Deprecated environment variable ${MODEL} found
+    ...
+    ERROR: Deprecated environment variable ${NEIGHBORHOOD_WIDTH} found
+    ERROR: Please set values that differ from the defaults in a METplus config file and unset GRID_STAT_CONFIG_FILE to use the wrapped MET config that is provided with the METplus wrappers.
+    ERROR: Compare values set in /path/to/GridStatConfig_trey to /path/to/met/share/met/config/GridStatConfig_default
+
+The easiest approach for investigating differences between two files is to use
+a visual diff tool that displays the files side-by-side and highlights any
+differences.
+Alternatively, the **diff** command is available on most Linux systems and can
+be used to quickly view line-by-line differences.
+However, viewing the actual files directly may still be necessary
+to see the context of the differences within the files.
+The **-y** argument can be provided to **diff** to view the differences in the
+terminal side-by-side in two columns.
+
+Please create a
+`METplus GitHub Discussions <https://github.com/dtcenter/METplus/discussions>`_
+post for any questions or clarification.
+
+The following examples of differences are shown using the format that is output
+by the **diff** utility.
+Lines that begin with the **<** character are from the first file passed
```

Comment:
Can this line be altered slightly? Maybe to 

Lines that begin with the **<** character are from the first file passed to **diff** (i.e. the user-defined wrapped MET config file).

or 

Lines that begin with the **<** character are from the first file passed to **diff**; in this case, the user-defined wrapped MET config file.

The same logic should apply to the next sentence.